src/web_interface/templates/system_health.html

```diff
@@ -73,21 +73,37 @@ <h3>{{component['name']}} machine status</h3>
                 <div class="col-sm-6 col-md-4 col-lg-3">
                     <h3>{{ component['name'] }} workload</h3>
                     <table class="table table-responsive-md table-hover table-bordered">
+                        {% set current_analyses = component['analysis']['current_analyses'] %}
+                        <tr>
+                            <th class="table-head-light" colspan=3 style="text-align: center; padding:5px; color: #495057">Currently Analyzed Firmware</th>
+                        </tr>
+                        {% if current_analyses %}
```

Comment:
We could make the complete block optional. So drop the Empty and move the if above the "Currently Analyzed Firmware" heading.

Reply:
if you think that's better ... changed